src/resources/projects/website/listing/quarto-listing.js

```diff
@@ -66,9 +81,14 @@ window.document.addEventListener("DOMContentLoaded", function (_event) {
     const category = decodeURIComponent(
       atob(categoryEl.getAttribute("data-category"))
     );
-    categoryEl.onclick = () => {
+    categoryEl.onclick = (e) => {
+      // Allow holding Ctrl/Cmd key for multiple selection
+      // Clear other selections if not using Ctrl/Cmd
+      if (!e.ctrlKey && !e.metaKey) {
+        selectedCategories.clear();
+      }
```

Comment:
I don't know if that is a good idea as it makes the multiple selection feature "hidden".
I thought this was less disruptive than checkboxes everywhere.
This was the only idea I came up with to keep previous visual appearance.